averages.js

```diff
@@ -0,0 +1,51 @@
+var _ = require("lodash");
+var pry = require('pryjs');
+
+function total(array) {
+	total = 0;
+	for(let i = 0; i < array.length; i++) {
+		total += array[i]
+	}
+	return total;
+};
+
+function mean(array) {
+	total = 0;
+	for(let i = 0; i < array.length; i++) {
```

Comment:
Great job! Consider solutions that use `Array.prototype` functions like `map` and `reduce`.